index.html

```diff
@@ -43,11 +43,108 @@ <h1 class="prof">I am Software Developer</h1>
             <a href="#" class="down-button">
                 <img src="./image/Disabled.svg" alt="">
             </a>
-        
-
+        </div>
+    </section>
 
+     <!-- content 2 -->
+    <section class="main2">
+        <div class="title">
+            <h1>My Recent Work</h1>
+            <div class="end-line"></div>
+        </div>
+        <div class="project-container">
+            <img src="./image/pic.jpeg" alt="">
+            <div class="desc">
+                <h1>Multi-Post Stories Gain+Glory</h1>
+                <ul>
+                    <li>HTML5</li>
+                    <li>HTML5</li>
+                    <li>HTML5</li>
+                </ul>
+                <button type="button">See project</button>
+            </div>
+        </div>
+        <div class="project-container">
+            <img src="./image/pic.jpeg" alt="">
+            <div class="desc">
+                <h1>Multi-Post Stories Gain+Glory</h1>
+                <ul >
+                    <li>Ruby on Rails</li>
+                    <li>HTML5</li>
+                    <li>Javascript</li>
+                </ul>
+                <button type="button">See project</button>
+            </div>
         </div>
+        <div class="project-container">
+            <img src="./image/pic.jpeg" alt="">
+            <div class="desc">
+                <h1>Multi-Post Stories Gain+Glory</h1>
+                <ul>
+                    <li>Ruby on Rails</li>
+                    <li>HTML5</li>
+                    <li>Javascript</li>
+                </ul>
+                <button type="button">See project</button>
+            </div>
+        </div>
+        <div class="project-container">
+            <img src="./image/pic.jpeg" alt="">
+            <div class="desc">
+                <h1>Multi-Post Stories Gain+Glory</h1>
+                <ul>
+                    <li>Ruby on Rails</li>
+                    <li>HTML5</li>
+                    <li>Javascript</li>
+                </ul>
+                <button type="button">See project</button>
+            </div>
+        </div>
+    </section>
+    <!-- content 2 -->
+    <section class="main3">
+        <h1>About me</h1>
```

Comment:
- [ ] let's make sure we add a background image to match the Figma design  

yours | Figma Design
:--:|:--:
![image](https://user-images.githubusercontent.com/87689547/148622489-a2f2821b-ec2a-4a99-bfd3-c04f8113e5ef.png) | ![image](https://user-images.githubusercontent.com/87689547/148622522-b26201ae-b1e2-48fb-8a81-d8595e50f671.png)